detections/web/crushftp_authentication_bypass_exploitation.yml

```diff
@@ -0,0 +1,57 @@
+name: CrushFTP Authentication Bypass Exploitation
+id: 82eb7f64-d219-4e21-acfe-956de84c1a35
+version: 1
+date: '2025-04-08'
+author: Michael Haag, Splunk
+status: production
+type: TTP
+description: The following analytic detects potential exploitation of the CrushFTP authentication bypass vulnerability (CVE-2025-31161). This detection identifies suspicious command execution patterns associated with exploitation of this vulnerability, such as executing mesch.exe with specific arguments like b64exec or fullinstall. This activity is indicative of an attacker exploiting CVE-2025-31161 to gain unauthorized access to the CrushFTP server and perform post-exploitation activities.
+data_source:
+- CrushFTP
```

Comment:
@MHaggis Can we add some output_fields to the Crush FTP data source file? 

Example: https://github.com/splunk/security_content/blob/1a084095d53087f2535894becfefd668ecc8db9f/data_sources/sysmon_eventid_1.yml#L120

These fields is the minimum number of basic fields needed in an output of a detection.

Also, lets try and use CIM names so -> user instead of username , process instead of cmd?